wildfly-modules/jboss/container/wildfly/startup/artifacts/bin/openshift-launch.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "`date "+%Y-%m-%d %H:%M:%S"` EAP Image on OpenShift launch"
```

Comment:
The image in this repository is the WildFly image. The EAP images are defined in: https://github.com/jfdenise/jboss-eap-modules
It seems that this PR could be closed there and opened against EAP. Thank-you.